### PR TITLE
feat: build main functions, add unit test and add readme

### PR DESCRIPTION
**Pull request by Nekotoxin**
Signed-off-by: Nekotoxin <Nekotoxin@outlook.com>

Some features from the golang watcher do not apply to the python version

### CODE REVIEW

**Comment by Nekotoxin** [comment]
@leeqvip plz review

File: README.md [hunk 1]
```diff
@@ -1,2 +1,50 @@
 # redis-watcher
```
Comment (hsluoyz) [inline README.md:1]: duplicated title

File: casbin_redis_watcher/__init__.py [hunk 2]
```diff
@@ -1 +1,2 @@
-from . import options, util, watcher
\ No newline at end of file
+from .options import WatcherOptions
```
Comment (hsluoyz) [inline casbin_redis_watcher/__init__.py:1]: Add license headers for all source code files

**Comment by Nekotoxin** [comment]
@leeqvip plz review

**Comment by hsluoyz** [comment]
@Nekotoxin you can ping leeqvip in QQ or WeChat for faster response

**Comment by Nekotoxin** [comment]
> @Nekotoxin you can ping leeqvip in QQ or WeChat for faster response

Thanks, I've contacted him

**Review by leeqvip — APPROVED** [review]

**Pull request merged**